src/tools/schema2kotlin.ts

```diff
@@ -148,6 +149,8 @@ class KotlinGenerator implements ClassGenerator {
     this.encode.push(`${fixed}.let { encoder.encode("${field}:${typeChar}", ${fixed}) }`);
 
     this.fieldSerializes.push(`"${field}" to ${fixed}.toReferencable()`);
+
+    this.fieldsForToString.push(`${fixed} = $${fixed}`);
```

Comment:
Is this supposed to be a double dollar sign?

Reply:
Yep! The output of toString looks like this:

```
override fun toString() = "Foo(bar = $bar, baz = $baz)"
```

Reply:
Ahh!